lib/linux_netns.c

```diff
@@ -43,9 +43,8 @@
 DEFINE_MTYPE_STATIC(LIB, NS, "NetNS Context")
```

Comment:
So typically we have used <system>_<subsystem>.c for file names.  Why is this different?  Shouldn't this be ns_linux.c?  Why are we deviating?

Reply:
maybe, I did not think of it enough. I have been proposed to move from ns to netns ( with Renato).
Would netns_linux.[ch] be ok  ?